corehq/apps/hqwebapp/templates/hqwebapp/maintenance_alerts.html

```diff
@@ -166,26 +166,20 @@ <h3 class="panel-title">
                           class="btn btn-primary"
                           name="command"
                           value="activate"
-                          data-bind="visible: status === 'inactive'">
-                    {% trans "Activate Alert" %}
+                          data-bind="visible: !active && !expired">
+                    <span data-bind="visible: !!start_time || !!end_time">{% trans "Schedule Alert" %}</span>
+                    <span data-bind="hidden: !!start_time || !!end_time">{% trans "Activate Alert" %}</span>
```

Comment:
I don't love the similar-but-different logic here but I don't know that there's a neat if/else I can do. The different text isn't critical, just a ui nicety.

Reply:
you could add `scheduled` or `has_schedule` back as a computed field in the JS model or the view (like with `expired`) and reference that.  I don't think it's that big a deal, though.